rfc(0009): correct GTK/Android status rows; note A2 report-only divergence (#26)
dasher-project/governance: rfcs/0009-crash-reporting.md

| Dasher-Windows | Implemented | Reference implementation: `AppDomain.UnhandledException`, crash file, PII scrubber with tests. |
| Dasher-Apple (macOS) | Implemented | `NSSetUncaughtExceptionHandler` plus an alive-marker unclean-shutdown detector. PostHog is the sole channel (macOS has no TestFlight). |
| Dasher-Apple (iOS, visionOS) | Not started in production | Betas lean on TestFlight. The PostHog `$exception` path for production builds is pending. Caught engine errors are reported through `captureEngineError` on all Apple targets. |

> | Dasher-GTK | Implemented | `std::set_terminate` + SIGSEGV/SIGABRT/SIGILL handlers writing a synchronous crash file; PII scrubber; PostHog `$exception` on next launch. Native signal shim coverage of engine faults landed with the v0.2.2 work ([Dasher-GTK #41](https://github.com/dasher-project/Dasher-GTK/pull/41)); sticky engine-error reporting per Amendment A2 below. |

Reviewer: Click-appliable version of the correction from my comment. Signal handling landed in `1c7f4d9` (2026-07-18), a month before #41, and #41 adds no signal-handling code at all (`grep -cE '\+.*(SIGSEGV|SIGABRT|SIGILL|sigaction)'` over its diff returns 0) — it adds sticky-flag polling, which is the other mechanism.

```suggestion
| Dasher-GTK | Implemented | `std::set_terminate` + SIGSEGV/SIGABRT/SIGILL handlers writing a synchronous crash file, PII scrubber, and PostHog `$exception` flush on next launch — all landed in `1c7f4d9` (2026-07-18). Sticky engine-error reporting per Amendment A2 below added in [Dasher-GTK #41](https://github.com/dasher-project/Dasher-GTK/pull/41). |
```

  calling frame, surface a message, then `dasher_destroy` + `dasher_create` (or
  ask the user to relaunch). `dasher_reset` does **not** clear it; only engine
  recreation does.
  *Platform divergence (August 2026):* the shipped frontends implement the

>   **detection and reporting** half — the flag is polled once per frame and

Reviewer: Minor precision, take it or leave it: GTK short-circuits on `!m_engine_error_reported`, so the flag stops being polled once it has fired rather than being polled for the life of the session.

```suggestion
  **detection and reporting** half — the flag is polled each frame until it fires and
```
